api/src/core/adapters/wikidata/getWikidataSoftware.ts

```diff
@@ -58,6 +87,23 @@ export const getWikidataSoftware: GetSoftwareExternalData = memoize(
             return { "label": entity.aliases.en?.[0]?.value, "id": licenseId };
         })();
 
+        const { entity: programmingLanguageEntity } = await fetchEntity(
+            getClaimDataValue<"wikibase-entityid">("P277")[0]?.id
+        ).catch(() => ({ "entity": undefined }));
+        const programmingLanguageLabel = programmingLanguageEntity
+            ? wikidataSingleLocalizedStringToLocalizedString(programmingLanguageEntity.labels)
+            : undefined;
+        const programmingLanguageString = programmingLanguageLabel
+            ? resolveLocalizedString(programmingLanguageLabel)
+            : undefined;
+
+        const versionClaim = lastestVersionClaim(entity);
+
+        const publicationTimeDateValue = <WikidataTime>versionClaim?.qualifiers?.["P577"]?.[0].datavalue.value;
```

Comment:
Pour info, c'est sans doute identique à ça :
```suggestion
        const publicationTimeDateValue = versionClaim?.qualifiers?.["P577"]?.[0].datavalue.value as WikidataTime | undefined
```